engines/sci/sci.cpp

```diff
@@ -436,7 +436,21 @@ Common::Error SciEngine::run() {
 	}
 
 	if (getGameId() == GID_GK2 && ConfMan.getBool("subtitles") && !_resMan->testResource(ResourceId(kResourceTypeSync, 10))) {
-		suggestDownloadGK2SubTitlesPatch();
+		suggestDownloadPatch(_("GK2 has a fan made subtitles, available thanks to the good persons at SierraHelp.\n\n"
+			"Installation:\n"
+			"- download http://www.sierrahelp.com/Files/Patches/GabrielKnight/GK2Subtitles.zip\n"
```

Comment:
Does ScummVM allow us to make this a hyperlink instead of a bare URL? So it would say something like

> - download [the subtitles](http://www.sierrahelp.com/Files/Patches/GabrielKnight/GK2Subtitles.zip)